utils/verifySecretKey.ts

```diff
@@ -0,0 +1,7 @@
+import config from '../config'
+
+const verifySecretKey = async (secret: string): Promise<void> => {
+  if (secret !== config.get('secret.key')) throw Error('Credential is invalid')
```

Comment:
_Credential is invalid_ .. maybe this same with upper coment .. 👍

Reply:
update dude @sandisunandar99 :smile:

Reply:
mantappss boys,, wkwkwk